src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -321,10 +318,15 @@ public List<JobModel> enumerateJobs(
   }
 
   private List<FlightState> getAccessibleFlights(
-      int offset, int limit, FlightFilter filter, AuthenticatedUserRequest userReq)
+      int offset,
+      int limit,
+      FlightFilter filter,
+      AuthenticatedUserRequest userReq,
+      Set<String> jobIdSet)
       throws InterruptedException {
     int start = 0;
     int end = offset + limit;
+    boolean canListAnyJob = checkUserCanListAnyJob(userReq);
     HashMap<String, List<String>> roleMap = new HashMap<>();
     ArrayList<FlightState> flightStateList = new ArrayList<>();
```

Comment:
These should use the generic types too
```suggestion
    Map<String, List<String>> roleMap = new HashMap<>();
    List<FlightState> flightStateList = new ArrayList<>();
```